### DISCUSSION TITLE
How to use LinkPreviewOptions?

### DISCUSSION THREAD

**Discussion by Auximen** [windkh/node-red-contrib-telegrambot#374] [Q&A] [answered]
Hi, I can't use the new Telegram API element LinkPreviewOptions (https://core.telegram.org/bots/api#linkpreviewoptions) in node-red-contrib-telegrambot. Could you please tell me how to use LinkPreviewOptions in NodeRed? Thanks

**Comment by windkh** [comment]
I will check ...

**Comment by windkh** [comment, marked as answer]
Looking at the documentation: the linkpreviewoptions are an optional parameter. Therefore you need to add a property to 
msg.payload.options --> msg.payload.options.linkpreviewoptions = { is_disabled : false, ... }

**Comment by Aleem28** [comment]
hello mate .. iam new in github and i have no idea of how to use these files. I want to download some files in rapidgator but the time and speed restriction prevent me to do so
can you do a video on youtube explaing step by step how to install rapidgator downloder. How to use python script. I found few links for rapidgator but all files are in .py format I don't know how to use or run. Can anyone explain please.

**Reply by windkh** [reply]
??? I guess you are in the wrong thread

What is rapidgator?
